core/metacling/src/TCling.h

```diff
@@ -111,6 +111,7 @@ class TCling : public TInterpreter {
    TObjArray*      fRootmapFiles;     // Loaded rootmap files.
    Bool_t          fLockProcessLine;  // True if ProcessLine should lock gInterpreterMutex.
    Bool_t          fAllowLibLoad;     // True if library load is allowed (i.e. not in rootcling)
+   Bool_t          fHasCxxModule;     // True if C++ modules was enabled
```

Comment:
Could you rename to `fCxxModulesEnabled = false` and just flip the flag if `R__USE_CXXMODULES` is set or `getenv("ROOT_MODULES") != nullptr`?

Reply:
What's the problem with fHasCxxModule ?

Reply:
It has no meaning to me. What is has cxxmodule? Does it mean it was built or ROOT is intending to use it? Also the name is inconsistent to the comment next to it ;)